eng/tools/oav-runner/src/cli.ts

```diff
@@ -28,6 +36,24 @@ export async function main() {
   // just need to resolve that here to make ts aware of it
   const targetDirectory = opts.targetDirectory as string;
 
+  let fileList: string[] | undefined = undefined;
+  if (opts.fileList !== undefined) {
+    const fileListPath = resolve(opts.fileList as string);
+    try {
+      const fs = await import('node:fs/promises');
+      const fileContent = await fs.readFile(fileListPath, { encoding: 'utf-8' });
+      fileList = fileContent.split('\n')
+        .map(line => line.trim())
+        .filter(line => line.length > 0);
+      console.log(`Loaded ${fileList.length} files from ${opts.fileList}`);
+    } catch (error) {
+      console.error(`Error reading file list from ${opts.fileList}: ${error instanceof Error ? error.message : String(error)}`);
```

Comment:
I think it's slightly better to wrap and re-throw, like we do here:

https://github.com/Azure/azure-rest-api-specs/blob/main/.github/shared/src/swagger.js#L79
https://github.com/Azure/azure-rest-api-specs/blob/main/.github/shared/src/spec-model-error.js